### PR TITLE
Add refactoring to remove method and the ones in subclasses

### LINKED ISSUE #4425: Missing refactoring to remove method and the ones in subclasses
Missing refactoring to remove method and the ones in subclasses
 
https://github.com/Lin777/Enjoliveur/issues/28

### PR DESCRIPTION
**Pull request by Lin777**
Link video: https://www.youtube.com/watch?v=-_MnoGxVyLQ

Fix #4425

### CODE REVIEW

**Comment by Ducasse** [comment]
Sounds good to me. Can you rename the as yet unclassified?

**Comment by Lin777** [comment]
Of course.

Done :)

File: src/Refactoring-Core/RBRemoveHierarchyMethodRefactoring.class.st [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+{ #category : #removing }
+RBRemoveHierarchyMethodRefactoring class >> removeMethods: selectorCollection from: aClass [
+	^self new removeMethods: selectorCollection from: aClass
+]
+
+{ #category : #transforming }
+RBRemoveHierarchyMethodRefactoring >> delete: selector in: aClass [
+	 (aClass realClass methodDict keys includes: selector) ifTrue: [
```
Comment (dionisiydk) [inline src/Refactoring-Core/RBRemoveHierarchyMethodRefactoring.class.st]: There is method #includesSelector:

Comment (Lin777) [reply src/Refactoring-Core/RBRemoveHierarchyMethodRefactoring.class.st]: Resolved :)

**Pull request merged**